bin/i18n/utils/sync_up_base.rb

```diff
@@ -0,0 +1,93 @@
+require_relative '../i18n_script_utils'
+require_relative '../metrics'
+require_relative '../utils/crowdin_client'
+
+module I18n
+  module Utils
+    class SyncUpBase
+      Config = Struct.new :crowdin_project, :source_paths, :ignore_paths, keyword_init: true
+
+      MAIN_CROWDIN_PROJECT = 'codeorg'.freeze
```

Comment:
Could you explain why setting the `MAIN_CROWDIN_PROJECT = 'codeorg'.freeze` in here?

Reply:
I'm not sure I understand the question, but It's not a setting, it's a constant variable that we also use to define a default config value in this class
https://github.com/code-dot-org/code-dot-org/blob/8012cb6285d69629ee3dfa887470b185ab6d280c/bin/i18n/utils/sync_up_base.rb#L12-L18